news.html

```diff
@@ -0,0 +1,25 @@
+<div class="news-content-container">
+    <div class="feat-content-card">
+        <a class="cardClick" target="_blank" rel="noopener noreferrer"
+           href="https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps-landing-page.html">
```

Comment:
One link to keep in mind if/when we move away from using the develop branch for the docs.

Reply:
oh right...we should probably just use stable